modules/json/src/main/java/com/devonfw/module/json/common/base/ObjectMapperFactory.java

```diff
@@ -136,6 +140,11 @@ public ObjectMapper createInstance() {
       mapper.setSubtypeResolver(subtypeResolver);
     }
 
+    mapper.setSerializationInclusion(Include.NON_NULL);
+    mapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
```

Comment:
We should at least add a comment above:
```
// Write legacy date/calendar as readable text instead of numeric value
// See https://fasterxml.github.io/jackson-databind/javadoc/2.6/com/fasterxml/jackson/databind/SerializationFeature.html#WRITE_DATES_AS_TIMESTAMPS
```

This is the only line that could be questioned. We could also remove this setting. However, java.util.Date/Calendar is legacy crap anyhow.